.rubocop/config.yml

```diff
@@ -3,6 +3,7 @@ AllCops:
   DisabledByDefault: true
   SuggestExtensions:
     rubocop-rake: false
+  TargetRubyVersion: 2.5
```

Comment:
Why is this the target ruby version?

Reply:
If I don't set this at all, `bundle exec rubocop`

```
redis-memo.gemspec:15:29: C: Gemspec/RequiredRubyVersion: required_ruby_version and TargetRubyVersion (2.7, which may be specified in .rubocop.yml) should be equal.
  s.required_ruby_version = '>= 2.5'
```

If I set it to a different number (ex 2.7)

```
redis-memo.gemspec:15:29: C: Gemspec/RequiredRubyVersion: required_ruby_version and TargetRubyVersion (2.7, which may be specified in .rubocop.yml) should be equal.
  s.required_ruby_version = '>= 2.5'
                            ^^^^^^^^
```

I do think we should probably bump the required version to 2.7, but thought that was out of scope of the PR. What do you think?